src/pages/settings.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const settings = () => {
```

Comment:
We recently made the change to make sure all components are PascalCased, so please capitalize first letters.